charts/streams-app/templates/scaled-object.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.autoscaling.enabled }}
+{{- $root := . -}}
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {{ template "streams-app.fullname" . }}-so
+spec:
+  scaleTargetRef:
+    name: {{ template "streams-app.fullname" . }}
+  pollingInterval: {{ .Values.autoscaling.pollingInterval }}
+  cooldownPeriod:  {{ .Values.autoscaling.cooldownPeriod }}
+  minReplicaCount: {{ .Values.autoscaling.minReplicas }}
+  maxReplicaCount: {{ .Values.autoscaling.maxReplicas }}
+  triggers:
+    {{- range .Values.streams.inputTopics }}
+    - type: kafka
+      metadata:
+        bootstrapServers: {{ $root.Values.streams.brokers }}
+        consumerGroup: {{ $root.Values.autoscaling.consumergroup }}
+        topic: {{ . | quote }}
+        lagThreshold: {{ $root.Values.autoscaling.lagThreshold | quote }}
+        offsetResetPolicy: {{ $root.Values.autoscaling.offsetResetPolicy }}
+    {{- end }}
+    {{- range .Values.autoscaling.topics }}
```

Comment:
Can you merge the topics first in order to loop only once?

Reply:
Do we eve need the `autoscaling.topics` then?

Reply:
Yes, there are internal topics automatically created by Kafka or created by us for repartitioning

Reply:
Only leaving a comment for merging for now, because `concat list1 list2` is not supported in helm 2